### DISCUSSION TITLE
App para timer público

### DISCUSSION THREAD

**Discussion by kandebonfim** [frontendbr/forum#341] [Dicas]
Galera, deixando aqui o link de um app que ajudei a desenvolver recentemente que pode ser muito útil pra quem trabalha remote e precisa ficar em sincronia com o resto do time em uma reunião ou pair programming:

[Cuckoo - A productivity timer for remote teams](http://cuckoo.team/).

Como só tem dev aqui. Seria massa ter feedback da usablidade do app. Enfim, o que vocês acharam?

---

Galera que curtir aí dá aquela divulgada básica! (estamos precisando de beta testers)

**Comment by felipefialho** [comment]
Show!

**Comment by guilhermejcgois** [comment]
Até o title muda, que daora!

Ali no "Your name" em Settings, não tem como deixar "visivel" que é possível editar o nome? Ou não era pra poder editar? Senti falta desse _feedback_ visual na interface, não foi natural saber que podia editar... Talvez um _highlight_ na div da input ou nela mesmo, dá a impressão que é apenas uma _label_.

Seria legal também um esquema parecido com o Slack, onde eu tenho vários _timers_, um para cada equipe ou projeto, não sei se consegui me expressar bem ou se isso não vai muito de acordo com a proposta inicial da ferramenta.

**Comment by kandebonfim** [comment]
@guilhermejcgois irado! A sidebar de settings precisa de uns tapinhas mesmo. Principalmente esse lance do input.

Sobre os diferentes timers não sei se peguei a ideia direito mas o que rola é usar diferentes urls para cada time/projeto/reunião. Pra trocar de sala é só ir pra cuckoo.team/sala-que-eu-quiser-nessa-bagaça e mandar ver! hahaha

**Comment by ghost** [comment]
Que foda! :+1: :smile:

**Comment by guilhermejcgois** [comment]
Saquei @kandebonfim hahah

Imagino que assim seja bem mais simples de manter haha Simples e fácil de usar, gostei, parabéns pela sacadas de vocês, vou tentar usar com meus grupos da universidade hehe

**Comment by brunooomelo** [comment]
desculpa minha ignorância, mas porque eu usaria um timer?

**Comment by felipefialho** [comment]
@brunooomelo Se você trabalhar remoto por exemplo, ou se quiser medir sua eficiencia em determinada tarefa.

**Comment by brunooomelo** [comment]
@LFeh como se fosse um pomodoro timer?

mas o app esta show de boa, testei e gostei. só senti falta de poder retirar alguém da sala kkkk

**Comment by alexsandro-xpt** [comment]
Legal medir o tempo e tal, mas não entendi qual o retorno em usa a ferramenta no dia a dia.... Pode contar qual cenário ela é interessante?

**Comment by andresantos-tech** [comment]
Pode ser uma boa pra quem usa Pomodoro
https://pt.wikipedia.org/wiki/T%C3%A9cnica_pomodoro ou alguma outra
técnica de gerenciamento de tempo, e talvez possa ter mais controle do que
um timer convencional por poder criar equipes e etc.

2016-10-28 12:10 GMT-02:00 Alexsandro Souza Pereira <
notifications@github.com>:

> Legal medir o tempo e tal, mas não entendi qual o retorno em usa a
> ferramenta no dia a dia.... Pode contar qual cenário ela é interessante?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/frontendbr/forum/issues/341#issuecomment-256929941,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/ABNIbqiOEDgYSEuS6-6y7yaOys7g0lyfks5q4gJZgaJpZM4KhYAi
> .

## 
- André O. Santos -

**Comment by lex0c** [comment]
@kandebonfim Show!
Mas achei um bug (ou não...) Criei uma sessão no chrome e iniciei, depois entrei no chrome em private e usei a sessão para add outro "membro", fui fazer o mesmo no firefox mas ele trava na solicitação do nome me possibilitando criar vários membros com o mesmo nome na mesma sessão.

Ambiente: 
- Linux/Ubuntu 14.04 LTS (64bit)
- Mozilla Firefox 49.0
- Google Chrome v54.0.2840.71
- Sessão: http://cuckoo.team/xxx

**Comment by kandebonfim** [comment]
Valeu, @lleocastro! Acho que já sei um bom caminho pra acabar com esse bug!

**Comment by renatocarvalho** [comment]
@alexsandro-xpt a utilidade é exatamente essa que o @andre-forweb explicou.

Vou detalhar aqui alguns use cases para o Cuckoo.

**Pair Programming (remoto ou não)**. É massa que as duas pessoas ficam por dentro do tempo. É bem útil também na hora do break, onde geral é notificado quando o break acabou.

**Uso do Pomodoro em Grupo (remoto ou não)**. A gente usa bastante aqui essa parada. Quando estamos fazendo brainstorming em grupo, a gente coloca time boxing pra tudo. É bem útil para manter o foco e geral sincronizado. Colocando uma foto da gente usando.

![14691996_1633644986933654_6162108261810567691_o](https://cloud.githubusercontent.com/assets/16294/19902608/970aba96-a052-11e6-930e-c1733c0a060c.jpg)

**Uso do Pomodoro normal**. Se você quiser usar sozinho, tbm funciona muito bem.

**Outras dinâmicas onde você precisa de um timer.** Teve gente que já usou cozinhar, projetar na TV para eventos com muitas pessoas.
